src/components/plots/linePlot.ts

```diff
@@ -6,7 +6,7 @@ export module Plot {
     private _hoverDetectionRadius = 15;
     private _hoverTarget: D3.Selection;
     private _defaultStrokeColor: string;
-
+    protected _interpolationMode: string | ((points: number[][]) => string) = "linear";
```

Comment:
Consider using an Interpolator type (or use this type on D3's typing).  Not a huge deal.